src/App.js

```diff
@@ -0,0 +1,53 @@
+import './App.css';
```

Comment:
Use index.js for each folder to export (Avoid repeating names on the imports) and imports from similar index.js file in one line→ clean code